irs-edc-client/src/test/java/org/eclipse/tractusx/irs/edc/client/policy/service/EdcPolicyDefinitionServiceTest.java

```diff
@@ -84,7 +84,10 @@ void testCreatePolicyDefinitionRequest() throws JsonProcessingException, JSONExc
         JSONAssert.assertEquals(objectMapper.writeValueAsString(request), """
                         {
                         	"@context": {
-                        		"odrl": "http://www.w3.org/ns/odrl/2/"
+                        		"odrl": "http://www.w3.org/ns/odrl/2/",
+                        		"@vocab":null,
+                        		"edc":null,
+                        		"cx-policy":null
```

Comment:
null values in the context are causing errors in EDC. Please exclude the null values.